src/main/webapp/app/entities/therapeutic-regime/detail/therapeutic-regime-detail.component.spec.ts

```diff
@@ -46,5 +55,28 @@ describe('Component Tests', () => {
         expect(comp.therapeuticRegime).toEqual(jasmine.objectContaining(null));
       });
     });
+    /*
+    describe('manageFeedback', () => {
+      it('Should enter in the managefeedback service', inject(
+        [],
+        fakeAsync(() => {
+          // GIVEN
+          const feedback = new Feedback();
+          feedback.entityName = EntityFeedback.THERAPEUTIC_REGIME;
+          feedback.entityId = comp.therapeuticRegime?.id;
+          feedback.thumb = true;
+
+          spyOn(service, 'manageFeedbackFromEntity').and.returnValue(of({}));
+
+          // WHEN
+          comp.manageFeedback(true);
+          tick();
+
+          // THEN
+          expect(service.manageFeedbackFromEntity(feedback)).toHaveBeenCalled();
+        })
+      ));
+    });
+*/
   });
```

Comment:
does it make sense to have this code commented?

Reply:
Removed